### PR TITLE
please review :)

### PR DESCRIPTION
**Pull request by ewhite1999**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
 # URL shortener
 
 Greetings reviewers!
+
+## Installation and Usage
```
Comment (sukhrajsinghb) [inline README.md:5]: very nice README

File: README.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+
+- clone the repo and `cd` into the folder
+- to start the server:
+  - `cd` into the backend folder
+  - run `pipenv install` and `pipenv install --dev`
+  - run `pipenv shell`
+  - run `pipenv run dev`
```
Comment (sukhrajsinghb) [inline README.md:12]: maybe this is an issue on my side but I followed all the steps and i got this error "/bin/bash: flask: command not found", either way try to test this out on your side to see if this is a real issue or just a me issue)

Dafé tried and his works so it turns out this is just a me issue.

Comment (ewhite1999) [reply README.md:12]: yeah I've never seen it before, maybe because it was set up on mac?

File: backend/app.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import exc
+import helpers
+
+app = Flask(__name__)
+CORS(app)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///urls.db'
+db = SQLAlchemy(app)
```
Comment (sukhrajsinghb) [inline backend/app.py:11]: some reason whenever we try to type in the long url, the long url returns but there is no short url for us to use and type into the search bar at the top of the browser for us to use.

File: README.md (hunk 2 above)
Comment (ewhite1999) [reply README.md:12]: Does it work if you switch the order of shell and install, I'm still not sure the way around it is supposed to be

File: backend/app.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
+from http import server
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import exc
```
Comment (sukhrajsinghb) [inline backend/app.py:5]: nothing wrong with this but im just curious what is exc?

Comment (ewhite1999) [reply backend/app.py:5]: it helps handle sqlalchemy errors

File: backend/conftest.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
+from app import app
+import pytest
```
Comment (sukhrajsinghb) [inline backend/conftest.py:2]: very nice testing

File: client/tailwind.config.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  content: ["./src/**/*.{js,jsx}"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+};
```
Comment (sukhrajsinghb) [inline client/tailwind.config.js:7]: 10/10 design
10/10 features
0/10 for Sukhraj's PC compatibility

**Pull request merged**